Assets/Scripts/Networking/PlayerManager.cs

```diff
@@ -230,6 +230,7 @@ public void Respawn()
             GetComponentInChildren<ApplyPostProcessing>().vignetteLayer.intensity.value = 0;
             fpsController.enabled = false;   //We disable the script so that we can teleport the player
             transform.position = respawnTransform.position;
+            GetComponent<FirstPersonController>().isPlayerInKeyLocZone = false;
```

Comment:
It's automatically false at start, so no need to set it to false here

Reply:
It's not false if a player dies in a zone and respawns.